src/components/add-site.tsx

```diff
@@ -117,16 +120,16 @@ export default function AddSite( { className }: AddSiteProps ) {
 						doesPathContainWordPress={ doesPathContainWordPress }
 					>
 						<div className="flex flex-row justify-end gap-x-5 mt-6">
-							<Button onClick={ closeModal } disabled={ isAddingSite } variant="tertiary">
+							<Button onClick={ closeModal } disabled={ isSiteAdding } variant="tertiary">
 								{ __( 'Cancel' ) }
 							</Button>
 							<Button
 								type="submit"
 								variant="primary"
-								isBusy={ isAddingSite }
-								disabled={ isAddingSite || !! error || ! siteName?.trim() }
+								isBusy={ isSiteAdding }
+								disabled={ isSiteAdding || !! error || ! siteName?.trim() }
```

Comment:
As we immediately close the dialog, do we need `isBusy` and `disabled` props?

Reply:
Yeah, it is for the case when the user might go and open the `Add site` modal from the sidebar while the new site is being added and a loading state is displayed so that they can't add another new site until this process is finished.
The form on the onboarding does not need it because we move away from that screen right into a loading screen.

Alternatively, if we want to allow creating new sites in parallel, we can remove the props.

For this PR, I left the existing flow and logic that is present in the app now (only one new site can be created at a time) but happy to adjust it.

Reply:
Thanks, it sounds reasonable.